app/routes.js

```diff
@@ -1,7 +1,9 @@
 const userController = require('./controllers/users_controller'),
+  authentication = require('../app/middlewares/authentication'),
   sessionController = require('./controllers/sessions_controller');
 
 exports.init = app => {
   app.post('/users', userController.signUp);
   app.post('/users/sessions', sessionController.logIn);
+  app.get('/users', [authentication.handle], userController.index);
```

Comment:
no need for it to be an array if you are only using one middleware, remember you can write this as 
`app.get('/users', authentication.handle, userController.index);`

Reply:
Cool